lib/webpacker/commands.rb

```diff
@@ -5,12 +5,34 @@ def initialize(webpacker)
     @webpacker = webpacker
   end
 
-  def clean(count = 2)
-    if config.public_output_path.exist? && config.public_manifest_path.exist? && versions.count > count
-      versions.drop(count).flat_map(&:last).each do |file|
-        File.delete(file) if File.file?(file)
-        logger.info "Removed #{file}"
-      end
+  # Cleanup old assets in the compile directory. By default it will
+  # keep the latest version, 2 backups created within the past hour.
```

Comment:
In what way is the age of the retained files important? This just causes asset availability downtime and doesn't include any explanation of why such a thing is desirable.

Reply:
@gauravtiwari @sbeam, maybe we can use the versions so long as they are OLDER than the age?

Right now, as soon as we find a few files with timestamps that differ by a second, we delete everything that's not in the current manifest.